### PR TITLE
Add Office Hours docs

### PR DESCRIPTION
**Pull request by dannytranlx**
I moved Stripe and Office Docs into their own `Add-ons` section.

### CODE REVIEW

File: source/includes/addons/_office_hours_adding.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+### Adding Office Hours to your messaging channels
+
+To enable the Office Hours add-on, you will need to select your preferred time zone, add your office hours and write your away message.
```
Comment (mspensieri) [inline source/includes/addons/_office_hours_adding.md]: add comma after "add your office hours"

File: source/includes/addons/_office_hours_adding.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+By default, the Smooch dashboard is going to try to guess your current time zone. This time zone is going to be used for your users' messages and your office hours in order to dispatch the message at the right moment.
+
+You will need to also provide the work hours where you will be available to reply; in other words, it's the moment where Smooch should not dispatch the away message. You can set different time ranges on different days or set it to _All day_.
+
+The away message will now be only dispatch when the user messaged you out of your work hours. This will give you the opportunity to tell your customer to come back later or message you through another channel.
```
Comment (mspensieri) [inline source/includes/addons/_office_hours_adding.md]: > The away message will only be dispatched when the user messages you outside of your work hours.

File: source/includes/addons/_office_hours_adding.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+The Office Hours add-on can also be set to reply to specific messaging channels if needed. If you select `iOS` and not `Web` for exemple, a user sending a message from a iOS device will prompt the away message, but not the user for the user on `Web`.
```
Comment (mspensieri) [inline source/includes/addons/_office_hours_adding.md]: > a user sending a message from an iOS device will prompt the away message, but a user on `Web` will not.

File: source/includes/addons/_office_hours_adding.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+<aside class="notice">There's a few exceptions that will avoid the message from being dispatched: when it's been already dispatched in that day, in the last 12 hours or you've been already messaging your user in the last 15 minutes.</aside>
```
Comment (mspensieri) [inline source/includes/addons/_office_hours_adding.md]: > There are a few exceptions that will prevent the message from being dispatched: if it's already been dispatched on that day, within the last 12 hours, or if you've messaged the user in the last 15 minutes.

**Comment by mspensieri** [comment]
:+1:

File: source/includes/addons/_office_hours_adding.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+To enable the Office Hours add-on, you will need to select your preferred time zone, add your office hours, and write your away message.
+
+By default, the Smooch dashboard is going to try to guess your current time zone. This time zone is going to be used for your users' messages and your office hours in order to dispatch the message at the right moment.
+
+You will need to also provide the work hours where you will be available to reply; in other words, it's the moment where Smooch should not dispatch the away message. You can set different time ranges on different days or set it to all day by selecting the 12:00 AM to 11:59 PM range.
+
+The away message will only be dispatched when the user messages you outside of your work hours. This will give you the opportunity to tell your customer to come back later or message you through another channel.
+
+The Office Hours add-on can also be set to reply to specific messaging channels if needed. If you select `iOS` and not `Web` for exemple, a user sending a message from an iOS device will prompt the away message, but a user on Web will not.
```
Comment (Mario54) [inline source/includes/addons/_office_hours_adding.md]: exemple -> example

**Comment by Mario54** [comment]
👍

File: source/includes/addons/_office_hours_adding.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+### Adding Office Hours to your messaging channels
+
+To enable the Office Hours add-on, you will need to select your preferred time zone, add your office hours, and write your away message.
+
+By default, the Smooch dashboard is going to try to guess your current time zone. This time zone is going to be used for your users' messages and your office hours in order to dispatch the message at the right moment.
+
+You will need to also provide the work hours where you will be available to reply; in other words, it's the moment where Smooch should not dispatch the away message. You can set different time ranges on different days or set it to all day by selecting the 12:00 AM to 11:59 PM range.
```
Comment (alavers) [inline source/includes/addons/_office_hours_adding.md]: Suggested edit:

You will need to also provide the work hours during you will be available to reply. In other words, you designate the time spans where Smooch _should not_ send an away message.

Comment (alavers) [inline source/includes/addons/_office_hours_adding.md]: Actually... I like the first sentence of the next paragraph better. It makes this one redundant. Suggest simply removing "in other words, it's the moment where Smooch should not dispatch the away message."

**Comment by alavers** [comment]
Rest looks :+1:

File: source/includes/addons/_office_hours_adding.md (hunk 6 above)
Comment (dannytranlx) [inline source/includes/addons/_office_hours_adding.md]: ...the work hours _during_ you will be available to reply?

Really? Sounds off 👀

Comment (alavers) [inline source/includes/addons/_office_hours_adding.md]: oops "during which"

Comment (dannytranlx) [inline source/includes/addons/_office_hours_adding.md]: oh right! thanks!

**Pull request merged**